react-frontend/src/App.js

```diff
@@ -50,7 +50,7 @@ function App() {
           <Route path="/listings/:listingID" element={<ListingView />} />
           <Route path="/listings/:listingID/buy" element={<ProtectedRoute> <BuyNow /></ProtectedRoute> } />
           <Route path="/listings/:listingID/offer" element={<ProtectedRoute> <MakeOfferPage /></ProtectedRoute> } />
-          <Route path="/listings/:listingID/chat" element={<ProtectedRoute> <ChatPage /></ProtectedRoute> } />
+          <Route path="/chat" element={<ProtectedRoute> <ChatPage /></ProtectedRoute> } />
```

Comment:
If we don't use listings/:listingID for the chat, should we use userID instead?

Reply:
we need to user listing id to get the seller's user id

Reply:
unless you have any other recommendations